src/js/components/deployments/deployment-wizard/review.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import pluralize from 'pluralize';
+
+import { Chip, List } from '@material-ui/core';
+import { InfoOutlined as InfoOutlinedIcon } from '@material-ui/icons';
+
+import ExpandableDeviceAttribute from '../../devices/expandable-device-attribute';
+import { getRemainderPercent } from '../../../helpers';
+
```

Comment:
to get the dates to further align with the design you could add this: `const dateOptions = { year: 'numeric', month: 'numeric', day: 'numeric', hour: 'numeric', minute: 'numeric' };` here...